compose.yaml

```diff
@@ -36,4 +36,4 @@ services:
     ports:
       - 26257:26257
       - 8080:8080
-    command: start-single-node --insecure --store=type=mem,size=0.25 --advertise-addr=localhost
```

Comment:
is a stateful node more useful here?

Reply:
hum, i removed that because i was running out of memory every time. Maybe i should try just increasing the limit

Reply:
worth a try!